python/lsst/validate/drp/matchreduce.py

```diff
@@ -154,6 +148,11 @@ def _loadAndMatchCatalogs(repo, dataIds, matchRadius,
         calibration.
     matchRadius :  afwGeom.Angle(), optional
         Radius for matching. Default is 1 arcsecond.
+    useJointCal : bool, optional
+        Use jointcal/meas_mosaic outputs to calibrate positions and fluxes.
+    skipTEx : bool, optional
+        Skip TEx calculations (useful for older catalogs that don't have
+        PsfShape measurements).
```

Comment:
Please put classes in back tickets at least for new docs, e.g. `bool` -> `` `bool` ``